tools/dmiAlphabetizer/readme.md

```diff
@@ -0,0 +1,23 @@
+# .dmi Alphabetical sorter
+Takes .dmi files and sorts them in place. See -h for information on use.
+You should REALLY think about resaving files after you use this program on it.
+When run from the cmd line it prints out every file that it touches, if a file is on that list
+it needs to be resaved so that the metadata is what byond expects. Now I know that it will run
+without resaving (byond's current version doesn't seem to care) but future versions may not.
+And for small files the file is usually bigger (especially paletted pngs, since they have a bug
+in the Pillow library afaik, or I'm incapable of the mental gymnastics to fix it) before you resave.
+also paths like ..\..\icons should work fine when used with the recurser while on windows.
+Mileage on other operating systems may vary.
+##Requirements
+So for this thing to work you need to install pillow for python 3. any flavor
+of python 3 should work. Ensure you don't have PIL installed (you should know already
+if you do).
```

Comment:
Requirements should be a easy to read format like a list or bullet points.

Requirements in complete sentences like this are hard to read and parse, especially when you come back after fulfilling one to see the next one.